.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Nope, no more Markdown issue templates :) let us use issue forms instead. I can make another PR and give you the whole credit. 🙂

Reply:
> Nope, no more Markdown issue templates :) let us use issue forms instead. I can make another PR and give you the whole credit. slightly_smiling_face

yep we have used that in other repos

Reply:
np I'll change them to issue template forms myself